### ISSUE TITLE
[req] Torrentleech.pl

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#7014]
id: torrentleech-pl
name: Torrentleech.pl
description: "Torrentleech.pl is a POLISH Private Torrent Tracker for 0DAY / GENERAL"
type: private
links:
  - https://torrentleech.pl/

**Comment by garfield69** [comment]
jackett 0.12.1732

**Comment by garfield69** [comment]
site is suspending service due to lack of funds to pay for server.
![image](https://user-images.githubusercontent.com/6933240/76294822-71554500-6318-11ea-8dd2-7afba904df77.png)

**Comment by RoloSoze** [comment]
Yeah, it's not gone forever. I'd keep it. They might come back once they sort out their financial issues.

**Comment by garfield69** [comment]
back online.

**Comment by ilike2burnthing** [comment]
https://torrentleech.pl/ - Connection timed out

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
- https://torrentleech.pl/ > aftgermarket.pl domain for sale page OR 429 Too Many Requests OR 403 Forbidden
- redirects to https://playhost.pl/ a gaming server provider

**Comment by garfield69** [comment]
v0.24.2277
